src/test/java/com/customer/customerService/CrediCardApplicationTest.java

```diff
@@ -0,0 +1,166 @@
+package com.customer.customerService;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.customer.model.CreditCard;
+import com.customer.repository.CreditCardRepository;
+import com.customer.service.CreditCardServiceImpl;
+
+
+
+public class CrediCardApplicationTest {
```

Comment:
soo many blank lines on top... limit it to 1

Reply:
Done.